core/src/main/java/com/yahoo/oak/Chunk.java

```diff
@@ -137,7 +146,7 @@ void readKeyFromEntryIndex(ThreadContext ctx) {
     /**
      * See {@code EntrySet.readValue(ThreadContext)} for more information
      */
-    void readValueFromEntryIndex(ThreadContext ctx) {
+    void readValue(ThreadContext ctx) {
```

Comment:
You were the one that suggested NOT naming this method `readValue()` because it was confusing.

Reply:
I still think all those similar names in Chunk interface are confusing. But it is more reasonable to to have readValueFromEntryIndex when indeed some index is supplied (only when it is something out of context). Otherwise just readValue. I would do the same for the key.